README.md

```diff
@@ -1,2 +1,13 @@
-# I-D-with-yang-template
-A template repository for Internet-Draft with YANG Modules management
+# "Internet-Draft with YANG Modules" Template Repository
+
+Use this repository as a template if you want to start working on
+[IETF](https://www.ietf.org/) documents with YANG Modules.
```

Comment:
This is currently the same README as https://github.com/martinthomson/internet-draft-template/blob/main/README.md 
Shall both exist in parallel midterm? If yes, this README should specify what is different.

Reply:
What I meant was: When someone comes along both this README as well as https://github.com/martinthomson/internet-draft-template/blob/main/README.md which one should he clone?
Preferably ours since it is more "official" due to the IETF-OPS-AD? Preferably Martin's since it is the original? Ours when more automatisms are needed and Martin's otherwise? This is the information I would expect in the README. Not necessarily in the first commit if we want to clone unmodified (see other comment), but soon.

Reply:
The one that will be closed in ours. This part is modified as we are adding the required structure/placeholders for yang part.

Reply:
I think we can add a short note indicating that this repository has been cloned from https://github.com/martinthomson/internet-draft-template and enhanced to support IETF documents which include YANG models.

Reply:
Sure. Done https://github.com/IETF-OPS-AD/I-D-with-yang-template/pull/2/commits/8fe277d08c61584d109279d27027505969ebdf2e